test/jasmine/tests/hover_label_test.js

```diff
@@ -4831,6 +4831,76 @@ describe('hovermode: (x|y)unified', function() {
         .then(done, done.fail);
     });
 
+    it('should format differing position using *xother* `hovertemplate` and in respect to `xhoverformat`', function(done) {
+        Plotly.newPlot(gd, [{
+            type: 'bar',
+            hovertemplate: '%{y}%{_xother:.2f}',
```

Comment:
```suggestion
            hovertemplate: 'y(_x):%{y}%{_xother:.2f}',
```
Just to make it easier to read the test output - likewise I'd suggest including the underscores in traces 3 and 4